requirements.txt

```diff
@@ -26,17 +26,17 @@ ecdsa==0.13
 Fabric==1.10.1
 gunicorn==19.2.1
 html2text==2015.2.18
-html5lib==0.95
```

Comment:
iirc this was pinned because we're using older version of weasyprint, because we're using older version of something else. requires testing if it doesn't crash our pdf generation.

Reply:
As you can see in the warning (initial comment), the pinned version of weasyprint requires html5lib>=0.999.